djstripe/event_handlers.py

```diff
@@ -143,53 +142,31 @@ def other_object_webhook_handler(event):
 #
 
 
-class CrudType(object):
-	"""Helper object to determine CRUD-like event state."""
-
-	created = False
-	updated = False
-	deleted = False
-
-	def __init__(self, **kwargs):
-		"""Set attributes."""
-		for k, v in kwargs.items():
-			setattr(self, k, v)
-
-	@property
-	def valid(self):
-		"""Return True if this is a CRUD-like event."""
-		return self.created or self.updated or self.deleted
+class CrudType(Enum):
```

Comment:
@therefromhere would this make sense to move to [djstipe/enums.py](https://github.com/dj-stripe/dj-stripe/blob/master/djstripe/enums.py)?